lightapi/python/symbollightapi/connector/BasicConnector.py

```diff
@@ -46,3 +46,13 @@ async def put(self, url_path, request_payload, property_name=None):
 		return await self._dispatch('put', url_path, property_name, data=json.dumps(request_payload), headers={
 			'Content-Type': 'application/json'
 		})
+
+	async def post(self, url_path, request_payload, property_name=None):
+		"""
+		Initiates a POST to the specified path and returns the desired property.
+		Raises NodeException on connection or content failure.
+		"""
+
+		return await self._dispatch('post', url_path, property_name, data=json.dumps(request_payload), headers={
```

Comment:
you probably want to use `json=` instead of `data`:
a) it doesn't require json.dumps
b) you don't need headers as it does that automatically

apart from that, looks ok

Reply:
good to know, that is powerful! 👍🏼